### ISSUE TITLE
Hydra requires 'lv without declaring it to elpa

### ISSUE THREAD

**Issue by magnars** [abo-abo/hydra#323]
I would suggest adding `lv` to the list of required packages. :)

**Comment by wezzle** [comment]
@magnars i think this is breaking my spacemacs packages update right now. Do you perhaps know a quickifx? `File is missing: Cannot open load file, No such file or directory, lv`

**Comment by Juan-Cortez3** [comment]
Hi, @wezzle I tried to download the `lv.el` file in this package and put it in your load path. That works for me.

**Comment by wezzle** [comment]
@Juan-Cortez3 Thank you. I currently froze the package on an old version I had on a different computer, but that sounds better.

**Comment by magnars** [comment]
`M-x package-install` `lv` should do it :)

**Comment by lechten** [comment]
As @Juan-Cortez3 wrote, `lv.el` is part of this repository. So there is no need to declare it as required package. It should be included in tar balls on [MELPA](https://melpa.org/#/hydra), but isn't currently. No idea why.

**Comment by bmag** [comment]
[Recent change of Hydra's recipe](https://github.com/melpa/melpa/commit/82d9e1bfd501ed2d1cd11319c5e1eb17482e8922#diff-bb45a20c40958c7aac2020a41a846fb7) means `lv` is no longer bundled with `hydra` package, however this change was done without updating Hydra to depend on the standalone `lv` package, thus breaking Hydra (when installed via `package.el` from melpa).

**Comment by tarsius** [comment]
@abo-abo and everyone affected. Sorry about this!

While cleaning up some Melpa branches I noticed that I still had not merged a commit into `master` that I had written almost a year ago. That commit removed `rv.el` from they `hyrda` *package*. (Well it tried to do that. Actually it was broken and needed a fixup commit.)

I am fairly sure that I discussed the move of `lv` to a separate package with @abo-abo back when I wrote that commit it but can not find that discussion anymore. The idea was to allow other packages to use `lv` without having to depend on `hydra`. In a first step I added the `lv` package to Melpa but did not remove `lv.el` from `hydra` to ensure a smooth transition. Well that didn't play out so well...

I guess I did not get to the second step until now because @abo-abo never added the required dependency declaration. Then I forgot about this. Once I found the commit again I should have double checked that `hydra` depended on `lv` now and that everything else was in order before merging but failed to do so.
